container-storage-setup.sh

```diff
@@ -624,6 +624,15 @@ is_block_dev_partition() {
     return 0
   fi
 
+  # This will check if a loop/nvme device is passed and is already partitioned.
```

Comment:
Lets mention something like.

` For loop device partitions, lsblk reports type as "loop" and not "part". So check if device has a parent in the tree and if it does, there are high chances it is partition (except the case of lvm volumes)"